core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -982,6 +982,14 @@ public int getDefaultDecimalScale() {
         }
     }
 
+    public boolean enableHiveDdlQuote(){
+        return Boolean.parseBoolean(getOptional("kylin.source.hive.enable.quote", TRUE));
```

Comment:
Please follow the config naming convension, "kylin.source.hive.quote-enabled" is a better name, like "kylin.engine.spark.sanity-check-enabled"

Reply:
I have fix this.